style.css

```diff
@@ -0,0 +1,218 @@
+html {
+	font-size: 10px;
+	scroll-behavior: smooth;
+	font-family: 'Varela Round', sans-serif;
+	box-sizing: border-box;
+}
+
+.nav-bar {
+	position: relative;
+	display:flex;
+	align-items: center;
+	justify-content: center;
+	gap: 30px;
+	background-color: #BD4B4B;
+	padding: 20px;
+	flex-wrap: wrap;
+}
+
+.nav-bar::after {
+ 	 position: absolute;
+	  bottom: -50px;
+ 	 left: 0px;
+ 	 width: 100%;
+	  content:" ";
+ 	 background:
+ 	 radial-gradient(circle at 50% 0%, #BD4B4B 25%, #EEEEEE 26%, #EEEEEE 40%);
+ 	 background-color: #EEEEEE ;
+ 	 background-size: 30px 100px;
+ 	 height: 50px;
+}
+
+.cv-button {
+	background-color: #ffffff;
+	border-color: #B13126;
+	border-radius: 10px;
+	color: #000000;
+	font-weight: bold;
+	font-size: 3rem;
+	padding: 15px;
+}
+
+.navigation-links {
+	color: #ffffff;
+	text-decoration: none;
+	font-size: 3rem;
+}
+
+header {
+	background-color: #EEEEEE;
+	height: 90vh;
+	color: #000000;
+}
+
+header div span {
+	font-size: 6rem;
+	display:flex;
+	align-items: center;
+	justify-content: center;
+	text-align: center;
+	padding-top: 10%;
+	margin-bottom: 0;
+}
+
+.sparkle-inline {
+	width: 10% ;
+	display: inline;
+}
+
+header div h1 {
+	font-size: 11rem;
+	margin: 0;
+	display:flex;
+	align-items: center;
+	justify-content: center;
+	text-align: center;
+	padding-bottom: 150px;
+}
+
+.about-section {
+	background-color: #BD4B4B;
+	padding: 50px;
+	height: 100vh;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	gap: 50px;
+}
+
+.about-me-title {
+	padding-left: 30px;
+	font-size: 3rem;
+	color: #ffffff;
+}
+
+.about-me-text {
+	padding-left: 30px;
+	padding-bottom: 30px;
+	font-size: 2rem;
+	color: #ffffff;
+}
+
+.profile-photo {
+	width: 200px;
+	height: auto;
+	border-radius: 10%;
+}
+
+.portfolio-section {
+	background-color: #EEEEEE;
+	padding: 50px;
+	height: 100vh;
+}
+
+.portfolio-title {
+	display:flex;
+	align-items: center;
+	justify-content: center;
+	font-size: 3rem;
+	padding-top: 50px;
+	padding-bottom: 75px;
+	margin: 0;
+	height: auto;
+}
+
+.portfolio-parent {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	gap: 30px;
+	padding-top: 30px;
+	font-weight: bold;
+	color: #2a2e32;
+	font-size: 2rem;
+}
+
+.portfolio-item {
+	border-style: dotted;
+	border-color: #000000;
+	border-width: 5px;
+	width: 25%;
```

Comment:
instead of using this to ensure that they're all the same size, it's better to use CSS grid, that way they will all be the same width and height and it will be dynamic. then you would have to override it on mobile so it's no longer a horizontal grid (there are multiple ways of making it vertical)